packages/contracts/package.json

```diff
@@ -31,7 +31,9 @@
     "deploy:diamond": "ts-node ./scripts/deploy/manager/DiamondInAll.ts",
     "task": "ts-node ./scripts/task/task.ts",
     "clean": "forge clean",
-    "purge": "yarn clean"
+    "purge": "yarn clean",
+    "anvil": "ts-node ./scripts/anvil/anvil.ts",
+    "run deploy:development": "bash ./scripts/bash/2_DevelopmentDeploy.sh"
```

Comment:
Is this an issue to use the `bash` binary on windows?

Reply:
Might be. But who uses windows for dev work?

Reply:
Doing up a TS file instead cause I forgot about Windows users